### ISSUE TITLE
Comparison with int fails

### ISSUE THREAD

**Issue by baggepinnen** [JuliaMath/DoubleFloats.jl#104]
The following caused me a great deal of confusion..
```julia
debug> sum(a) |> typeof
Double64

debug> sum(a) ≈ 1
false

debug> sum(a) ≈ 1.0
true
```
There seems to be a method or conversion etc. missing
```
(@v1.4) pkg> st DoubleFloats
Status `~/.julia/environments/v1.4/Project.toml`
  [497a8b3b] DoubleFloats v1.1.6
```

**Comment by JeffreySarnoff** [comment]
thanks -- yes, conversion/promotion pair needs to be added .. on it

**Comment by JeffreySarnoff** [comment]
It turns out the problem was an out-of-date `isapprox` in this source.  Fixed in master and in v1.1.7 (pending merge)

**Comment by baggepinnen** [comment]
Great, sweet package :)
